documentation/docs/configuration.md

```diff
@@ -78,7 +78,8 @@ The following data will be sent to the endpoint if activated:
 * ErrorCode
 * ErrorCategory
 * CorrelationID (not hashed)
-* CommitHash
+* CommitId
+* CommitHash (Head commit hash of current build.)
```

Comment:
I guess this is pretty hard to understand ...

Reply:
Do you really need both values here? Maybe we can omit the one created for xmake.

Reply:
Yes, will remove the one for xMake.